redesign/src/styles/layout/_footer.scss

```diff
@@ -0,0 +1,93 @@
+footer {
+    height: 380px;
+    width: auto;
+    z-index: 3;
+
+    @media only screen and (min-width: 620px) {
+        display: flex;
+        flex-wrap: wrap;
+        height: 250px;
+        width: auto;
```

Comment:
Since you declare the width above, you shouldn't need to re-declare it here and in the next media query.